app/models/patient.rb

```diff
@@ -723,9 +723,20 @@ def latest_transfer
 
     # If a patient record has been inactive for 30 days or more
     # in the exposure workflow and is non-reporting
-    no_recent_activity = isolation_non_reporting
-                         .where('updated_at <= ?', 30.days.ago)
-                         .or(exposure_non_reporting.where('updated_at <= ?', 30.days.ago))
```

Comment:
This comment should probably be moved to be with the new `no_recent_activity_exposure` variable